test/promisified/consumer/consumeMessages.spec.js

```diff
@@ -410,8 +417,24 @@ describe.each(cases)('Consumer - partitionsConsumedConcurrently = %s -', (partit
             topic: topicName,
             partitions: partitions,
         });
-        await consumer.connect();
+
+        /* Reconfigure producer and consumer in such a way that batches are likely
+         * to be small and we get multiple partitions in the cache at once.
+         * This is to reduce flakiness. */
+        producer = createProducer({}, {
+            'linger.ms': 1,
+        });
+
+        consumer = createConsumer({
+            'groupId': groupId,
+        }, {
+            'fetch.message.max.bytes': 1,
+            'fetch.max.bytes': 1000,
+            'message.max.bytes': 1000,
```

Comment:
We  can leave only this one to get a single batch
```suggestion
            'fetch.message.max.bytes': 1,
```